README.md

```diff
@@ -12,12 +12,14 @@ Image for php-fpm. It is based on Alpine linux and thats why it is very small (~
 * PHP 7.1.3
 
 ## Running
-```
+
+```sh
 docker run --rm -p 9000:9000 -v /path/of/application:/application yavin/alpine-php-fpm:7.1
 ```
 
-Fallowing nginx configuration allow to connect to this FPM setup:
-<pre>
+Following nginx configuration allow to connect to this FPM setup:
+
+```nginx
```

Comment:
unfortunately formatting (`<b>` below) doesn't work with triple back-tick. Only with `<pre>`. Please revert this one change.

Reply:
But then it doesn't format well at all here:
https://hub.docker.com/r/yavin/alpine-php-fpm/
Is that `<b>` tag worth that?

Reply:
I didn't check description on hub. At first I tried to change description there, but unfortunately it gets updated on every build. Formatting there uhm ... could lokk better. Store is [little better](https://store.docker.com/community/images/yavin/alpine-php-fpm) but still doesn't support `<pre>`. 

I just remove formatting as you said. Thanks for help.